### ISSUE TITLE
Debian & Ubuntu maintainers for tes3mp

### ISSUE THREAD

**Issue by Koncord** [TES3MP/TES3MP#152]
We are looking for [Launchpad](https://launchpad.net/~tes3mp) maintainer for our project.

**Comment by IriaSomobu** [comment]
What is the problem? What do we have to do?

**Comment by Yaksinikos** [comment]
>What is the problem? What do we have to do?

I think to manage the ppa or so ... like the one of openmm!
I am using Linux Mint :smile:!

**Comment by IriaSomobu** [comment]
@Translator5 I never maintained ppa so I need to know what I should do.

> I am using Linux Mint 😄!

Good.

**Comment by Yaksinikos** [comment]
@iillyyaa2033 

> I never maintained ppa ...

Me too :rofl:

**Comment by IriaSomobu** [comment]
@Translator5 so I've asked:
> What is the problem? What do we have to do?

**Comment by Shnatsel** [comment]
I could do that provided I can get TES3MP to build in the first place. I've maintained plenty of PPAs.

But I'm rather busy with other stuff, so if anyone wants to take this over, feel free

**Comment by Yaksinikos** [comment]
Is it possible to get multiple administrators?

**Comment by Shnatsel** [comment]
Yes. Create a team and then create the PPA on the team. The team admin will be able to add and remove people at will.

**Comment by Yaksinikos** [comment]
As far the ppa is not on stage - why there is no Linux release of 6.0? ...

I would like to get into the maintainer team ;) 
did contact admins on launchpad!
